tutorial/config/task/random_small__latent.yaml

```diff
@@ -4,15 +4,26 @@ defaults:
 batch_size: 10
 
 feature_names:
-  - drugs_20
-  - metagenomics_174
-  - proteomics_195
+  - proteomics_1
+  - proteomics_2
+  - proteomics_3
+  - proteomics_4
+  - proteomics_5
+  - metagenomics_1
+  - metagenomics_4
+  - metagenomics_5
+  - metagenomics_10
+  #- drugs_20
+  #- metagenomics_174
+  #- proteomics_195
 
 model:
   num_hidden:
-    - 1000
-  num_latent: 150
+    - 12
+  num_latent: 8 #150
+  beta: .0001
```

Comment:
The changes to these config files (all the `random_small_`) need to be rolled back. I believe they were meant for the `random_continuous` dataset only.

Reply:
or just for testing? But I agree that this should be set via the command line for testing